insights/parsers/md5check.py

```diff
@@ -7,61 +7,84 @@
 The name and md5 checksums of specified files are stored as lists. Each
 entry will be processed by the shared parser.
 
-This MD5CheckSum class provide the following properties:
+This MD5CheckSum class provide the following attributes:
 
-* ``filename``: the file name which derived from the output.
-* ``md5sum``: the checksums for the file.
+* ``files``: the files which derived from the output.
+* ``checksums``: the checksums for the files.
 """
+
 from .. import parser, CommandParser
 from insights.specs import Specs
+from insights.parsers import ParseException
 
 
-@parser(Specs.prelink_orig_md5)
 class MD5CheckSum(CommandParser):
     """
-    Class to parse ``prelink -y --md5`` or ``md5sum`` command information.
+    Common class to parse ``prelink -y --md5`` or ``md5sum`` command information.
```

Comment:
Should the `prelink -y --md5` be removed as well?

Reply:
I would think currently not, because the `prelink` command is not usable due to framework's limitation and we indeed have plugin depends on this parser.